src/SpellsReference/Data/Context.cs

```diff
@@ -15,6 +15,16 @@ public class Context : DbContext, IContext
         protected override void OnModelCreating(DbModelBuilder modelBuilder)
         {
             modelBuilder.Conventions.Remove<PluralizingTableNameConvention>();
+
+            modelBuilder.Entity<Spellbook>()
+            .HasMany<Spell>(sb => sb.Spells)
+            .WithMany(s => s.Spellbooks)
+            .Map(cs =>
+            {
+                cs.MapLeftKey("SpellbookId");
+                cs.MapRightKey("SpellId");
+                cs.ToTable("SpellSpellbook");
```

Comment:
This is a nitpick, but if the "left" side of this relationship is "Spellbook" and the "right" side is "Spell", then I'd expect this table to be named "SpellbookSpell".